packages/e2e/plugins/index.ts

```diff
@@ -30,6 +31,22 @@ module.exports = (on: Cypress.PluginEvents, config: Cypress.PluginConfig) => {
     await afterRunHook();
   });
 
+  on("task", {
+    deleteFolder(folderName) {
+      console.log("deleting folder %s", folderName);
+
+      return new Promise((resolve, reject) => {
+        // eslint-disable-next-line consistent-return
+        rmdir(folderName, { maxRetries: 10, recursive: true }, (err) => {
+          if (err && err.code !== "ENOENT") {
+            console.error(err);
+            return reject(err);
+          }
+          resolve(null);
+        });
+      });
+    },
+  });
```

Comment:
Let's keep the task handlers in their own plugin(s). For this use case, I suggest you make a `fsPlugin` and follow the `firebasePlugin` structure. 
Please note: This is not a generally accepted convention for cypress plugins, but it made a lot of sense to me when writing the first one (`firebasePlugin`) and I believe we should keep things consistent, as well as modular (not writing plugins directly in `plugins/index.ts`)